client_wrapper.py

```diff
@@ -0,0 +1,169 @@
+"""
+ TODO: 
+
+-For Michael:
+ 1: Change duplicate Class names that will be imported.
+
+ 2: Instead of importing every class from a module(Listener, WakeWordEngine, DemoAction)
+    we can just wrap them to one Super class, and import it for simplicity,
+    like that we can just use the prediction functions of the class instead of building everything
+    in this file
+
+3: Change line number (72) and (78) to appropriate inference functions from the Super classes discussed
+   above
+
+-Extras(for me):
+
+ 1: Bring a TTS module, probably pyttsx3 or gtts, pyttsx3 is more reliable, and not bound to length limitation 
+
+"""
+
+
+import json
+import multiprocessing
+
+from client import Client
+from client_config import *
+from taskmanager import TaskManager
+
+
+from VoiceAssistant.wakeword.engine import Listener, WakeWordEngine, DemoAction
+from VoiceAssistant.speechrecognition.engine import Listener, SpeechRecognitionEngine, DemoAction
+#from VoiceAssistant.speechsynthesis.engine import TTS
+
+
+# no idea about them, seems a bit complicated to me, just need a ww and sr object
+# something like this:
+ww = WakeWordEngine()
+sr = SpeechRecognitionEngine()
+
+# and then use some function/method that returns predicted value after a session
+# example: 
+#			predicted = ww.start_wakeword_loop() which will be 0 or 1
+#				if predicted:
+#					speech = sr.run_speech_recognition() which will return a string value (predicted text)
+#
+#					% now pass the 'speech' variable into client to get the result from server %
+#
+#					inferenced = self._ask_input(speech)
+#					tts.say(inferenced)
+
+
+tts = None # TTS class will go here instead of None
+
+
+loop_not_finished = True
+
+
+def loop_speech(iter_list):
+	for i in iter_list:
+		tts.say(i)
+	loop_not_finished = False
+
+
+
+class Wrapper(TaskManager, Client):
+	def __init__(self):
+		super().__init__(self)
+
+	def start_module(self):
+
+		while True:
+			pred = ww.start_wakeword_loop() # start prediction

```

Comment:
change the line with the wakeword prediction function that returns 1 if wake word predicted, or 0 if not